components/overview/ContentSection.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <article class="content-section">
+    <div class="content-section__overview">
+      <h2 class="copy__title">
+        {{ title }}
+      </h2>
+      <p class="content-section__description">
+        {{ description }}
+      </p>
+      <AppLink class="copy__link" :url="link.url">
+        {{ link.label }}
+      </AppLink>
+    </div>
+    <div class="content-section__detail">
+      <slot />
+    </div>
+  </article>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import AppLink from '~/components/ui/AppLink.vue'
+
+@Component({
+  components: { AppLink }
+})
+export default class extends Vue {
+  @Prop(String) title
+  @Prop(String) description
+  @Prop(Object) link
+}
+</script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
+
+.content-section {
+  width: 100%;
```

Comment:
`width` and `margin-bottom` are [external geometry according to BEM](https://en.bem.info/methodology/css/#external-geometry-and-positioning), can we move these to the consumer of the component?

Reply:
Good point! We should review all the code because we are not doing it properly everywhere